File: RemoveReoccuring.js
// Given a string as your inputs, delete any recorruring character

let s = "This is a sentence with some reoccuring characters in it";

let removeReoccuring = (s) => {
    let seen = [];
    let res = "";
    for (let i=0; i<s.length; i++) {
        if (!(seen.includes(s[i]))) {
            res = res + s[i];
            seen.push(s[i]);
        }
    }
    return res;
}

console.log(removeReoccuring(s));

File: RemoveReoccuring.py
#Given a string as your inputs, delete any recorruring character

s = "This is a sentence with some reoccuring characters in it"

def delete_reoccuring(s):
    res_string = ""
    char_dict = {}
    for char in s:
        if char not in char_dict.keys():
            char_dict[char] = 1
            res_string = res_string + char
        else:
            continue
    return res_string

print(delete_reoccuring(s))